### PR TITLE
dfcf 3.7.3 (new cask)

### PR DESCRIPTION
**Pull request by ghost**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/d/dfcf.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  livecheck do
+    url :url
+    strategy :extract_plist do |item|
+      item["com.emmac.mac"]&.short_version
+    end
+  end
+
+  depends_on macos: ">= :el_capitan"
+
```
Comment (p-linnane) [inline Casks/d/dfcf.rb:19]:
```suggestion

```

Comment (p-linnane) [reply Casks/d/dfcf.rb:19]: This can be removed since the application works on Yosemite and later, but Homebrew only runs on El Capitan and later.

Comment (Koeniy) [reply Casks/d/dfcf.rb:19]: Thanks for your time and changes. I previously thought I needed to write the minimum supported macos version of Homebrew.

**Review by p-linnane — APPROVED** [review]
Thanks!

**Pull request merged**